clientgui/DlgExclusiveApps.cpp

```diff
@@ -282,71 +282,62 @@ void CDlgExclusiveApps::AddToListBox(wxListBox * theListBox) {
     bool hostIsWin = false;
     bool isDuplicate;
     wxArrayString appNames;
-    wxChar *extension = wxT("");
     wxString errmsg;
     CMainDocument* pDoc = wxGetApp().GetDocument();
 
     wxASSERT(pDoc);
     wxASSERT(wxDynamicCast(pDoc, CMainDocument));
 
+    // ? why use these instead of ifdefs?
```

Comment:
I think the answer is 'because Manager could be run on one OS (e.g. Windows) and manage a client that run different OS (e.g. MacOS))

Reply:
This comment should be removed from source code because it could just confuse someone